core/integration/features/verified/LP-large-prob-trading-bug.feature

```diff
@@ -0,0 +1,105 @@
+Feature: Replicate LP getting distressed during continuous trading, check if penalty is implemented correctly
+
+  Background:
+
+    Given the log normal risk model named "log-normal-risk-model-1":
+      | risk aversion | tau | mu | r | sigma |
+      | 0.000001      | 0.1 | 0  | 0 | 1.0   |
+    # And the following assets are registered:
+    #   | id  | decimal places |
+    #   | USD | 18             |
+    And the fees configuration named "fees-config-1":
+      | maker fee | infrastructure fee |
+      | 0.004     | 0.001              |
+    And the price monitoring named "price-monitoring-1":
+      | horizon | probability | auction extension |
+      | 10000   | 0.99        | 300               |
+    And the markets:
+      | id        | quote name | asset | risk model              | margin calculator         | auction duration | fees          | price monitoring   | oracle config          |
+      | ETH/MAR22 | ETH        | USD   | log-normal-risk-model-1 | default-margin-calculator | 1                | fees-config-1 | default-none | default-eth-for-future |
+    And the parties deposit on asset's general account the following amount:
+      | party  | asset | amount    |
+      | party0 | USD   | 5000000000   |
+      | party1 | USD   | 10000000000 |
+      | party2 | USD   | 10000000000 |
+      | party3 | USD   | 10000000000 |
+
+    Given the following network parameters are set:
+      | name                                          | value |
+      | market.stake.target.timeWindow                | 24h   |
+      | market.stake.target.scalingFactor             | 1     |
+      | market.liquidity.bondPenaltyParameter         | 0.2   |
+      | market.liquidity.targetstake.triggering.ratio | 0.1   |
+      | market.liquidity.stakeToCcySiskas             | 1.0   |
+
+   And the average block duration is "1"
+
+   Scenario: LP vol seems wrong when a new best bit/ask price comes in
```

Comment:
What do we mean by this label? Seems wrong but is correct? I am assuming the test is passing if it's a PR.